src/components/screens/Service.tsx

```diff
@@ -32,7 +32,7 @@ const umlautSwitcher = (text: string) => {
 
   const replacedText = text.replace(UMLAUT_REGEX, (match: string) => umlautReplacements[match]);
 
-  return replacedText;
+  return replacedText?.replace('​', '');
```

Comment:
why do we replace `''` with `''`? what trick is this?

Reply:
I don't know the reason for this either. There was this code in the development about 2 years ago and since I don't know exactly why, I thought I'd add it here. 🤷‍♂️

Reply:
no hints in the commit message from back then?

Reply:
Aha. I found it.

<img width="649" alt="image" src="https://github.com/user-attachments/assets/e0c55eaa-b3d0-44c0-9df0-577bca028087">

Reply:
oh yeah! that's how we like it.
so did you copy the code or typed newly? needs to be copied to get the correct signs, right!?